packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -65,25 +63,20 @@ class PlotlyDivGraph extends CoreElement {
 
   static const String frameGraph = 'graph_frame_timeline';
 
-  // Data is adding to a ploting graph every second or every 20 frames of data
-  // collected.  Otherwise plotly will lag is displaying the data.
-  static const int frameChunking = 20;
```

Comment:
I've eliminated the frameChunking logic and switched to just a timer at 6fps which should be equivalent worst case to the draw per 20 frames logic.
This makes the typical case for a flutter UI look a lot more responsive. 
Eliminated the mutex code as it doesn't seem to be needed as the code in the mutex wasn't async.